CMake/external_fastdds.cmake

```diff
@@ -84,4 +97,6 @@ target_include_directories(dds INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINAR
 target_link_libraries(dds INTERFACE debug ${CMAKE_CURRENT_BINARY_DIR}/fastdds/fastdds_install/lib/${FASTDDS_DEBUG_TARGET_NAME}${CMAKE_STATIC_LIBRARY_SUFFIX})
 target_link_libraries(dds INTERFACE optimized ${CMAKE_CURRENT_BINARY_DIR}/fastdds/fastdds_install/lib/${FASTDDS_RELEASE_TARGET_NAME}${CMAKE_STATIC_LIBRARY_SUFFIX})
 
+add_definitions(-DRS2_USE_DDS)
```

Comment:
Wouldn't it be easier to remember just `BUILD_WITH_DDS`? Why define another name?

Reply:
I saw it several times at LRS, so I aligned to it.
But sure no need for a new name, will use the option name